admin/sections/fileLoader.tsx

```diff
@@ -26,6 +26,8 @@ export const FileLoaderConfig = () => {
   const loaderForm = useForm({
     resolver: zodResolver(FileLoaderSchema),
   });
+  const fieldsChanged =
+    Object.keys(loaderForm.formState.dirtyFields).length > 0;
```

Comment:
Why we can't use this condition in chat.tsx too?

Reply:
Because we're sharing the same form for model config and chat config but split it into two sections, that's why in chat.tsx we only show the Update button if only the fields in that section has changed